src/Mono.Android/Android.Runtime/AndroidRuntime.cs

```diff
@@ -704,10 +704,11 @@ internal void AddPeer (IJavaPeerable value, IntPtr handle, JniHandleOwnership tr
 			}
 
 			if (Logger.LogGlobalRef) {
-				RuntimeNativeMethods._monodroid_gref_log ("handle 0x" + handleField.ToString ("x") +
-						"; key_handle 0x" + hash.ToString ("x") +
-						": Java Type: `" + JNIEnv.GetClassNameFromInstance (handleField) + "`; " +
-						"MCW type: `" + value.GetType ().FullName + "`\n");
+				RuntimeNativeMethods._monodroid_gref_log (
+					FormattableString.Invariant ($"handle 0x{handleField:x}") +
```

Comment:
I suspect it'll be more efficient if we do all the string concat *within* `FormattableString.Invariant()`, vs. doing string concat *between* `FormattableString.Invariant()`.  Would be interesting to look at the resulting IL.

Reply:
I didn't figure out how to best do these multi-line ones, where we were trying to line wrap.

For this case, maybe I just make it one line, as something like this causes `CA1305`:
```csharp
FormattableString.Invariant($"Foo: {foo}" + $"Bar: {bar}")
```
It wants each instance of `$""` to be wrapped by `FormattableString.Invariant()`.